examples/top-bar/index.html

```diff
@@ -29,8 +29,11 @@
     <link rel="stylesheet" href="http://tntvis.github.io/tnt.tooltip/build/tnt.tooltip.css" type="text/css"/>
     <script src="http://tntvis.github.io/tnt.tooltip/build/tnt.tooltip.min.js" charset="utf-8"></script>
 
-    <script src="https://dessimozlab.github.io/iHam/iHam.js"></script>
-    <link rel="stylesheet" href="https://dessimozlab.github.io/iHam/iHam.css" type="text/css"/>
+    <!-- <script src="https://dessimozlab.github.io/iHam/iHam.js"></script>-->
```

Comment:
probably better to remove the comments in the examples to reduce noise (unless you want the commented code to be present as alternatives options)